WikiWatershed/model-my-watershed
# Create codepaths for GWLFE Hydrology & Water Quality compare view

## Overview

This PR creates conditional codepaths for the GWLFE Hydrology and Water Quality charts and tables.

It's prep work for creating the Hydrology Line Charts for #2909, but I'm putting it up as a separate PR because the changes are pre-requisites for working on the other compare view charts.

Connects #2909

### Demo

![screen shot 2018-08-14 at 11 38 08 am](https://user-images.githubusercontent.com/4165523/44101932-8b0edbae-9fb6-11e8-9aee-5512ab1f0a7d.png)

## Testing Instructions

- serve this branch, then visit the TR55 compare view and verify that it works exactly as it did before
- visit the GWLFE compare view and verify that it still works as before but that the error messages in the console have been replaced with the designed console.warn statements depicted above.
